.gitattributes

```diff
@@ -0,0 +1 @@
+*.go text eol=lf
```

Comment:
This one might be out of scope, at least the scope of the PR no?

If not please add a comment in the file about its purpose

Reply:
Indeed, it makes sense.

please comsider my point about adding an inline comment to explain it